javascript/ui/page/passwordsignup_test.js

```diff
@@ -112,12 +114,28 @@ function testInitialFocus_name() {
 }
 
 
+function testDisplayNameIsNotRendered() {
+  if (goog.userAgent.IE && !goog.userAgent.isDocumentModeOrHigher(9)) {
+    return;
+  }
+  component.dispose();
+  component = new firebaseui.auth.ui.page.PasswordSignUp(
+      null,
+      false,
+      goog.bind(
+          firebaseui.auth.ui.element.FormTestHelper.prototype.onSubmit,
+          formTestHelper));
+  component.render(root);
+  assertNull(component.getNameElement());
+}
+
+
 function testInitialFocus_newPassword() {
```

Comment:
Add tests for initial focus and focus propagation when no display name is required.